src/main/learnocaml_client.ml

```diff
@@ -154,6 +154,15 @@ module Args_exercises = struct
           $color_when $verbose )
 end
 
+module Args_fetch = struct
+  let id =
+    value & pos_all string [] & info [] ~docv:"EXERCISE_ID" ~doc:
+      "Exercise identifier. Can be repeated. \
+       If not present, all the exercises will be downloaded"
```

Comment:
Please add a final dot.

Reply:
Speaking of it, there is no consensus in the code, some of the descriptions have a final dot (`--version`, `--help`), but not the others (`--server`, `--token`, for example).

Reply:
I prefer a final dot everywhere then.